src/lib/Preview.js

```diff
@@ -1202,6 +1217,9 @@ class Preview extends EventEmitter {
      * @return {void}
      */
     triggerError(err) {
+        // Always log preview errors
+        this.logPreviewError(err);
```

Comment:
I think there's a chance of double logging here. Above you bound to the `VIEWER_EVENT.error` event, which you then call to `triggerError`. On line 966 of your patch (this is old code),  we also bind to the `error` event coming from the viewer and call `triggerError`. This would end up calling `logPreviewError` twice, one from `handleViewerEvents` and one from `triggerError`

Reply:
Aww hell. So, as it is, node requires that you bind to the 'error' event (listener #1), then we listen for it in `handleViewerEvents` (listener #2). The huge issue here is *when* #1 is bound can be *after* any setup errors occur. 

Does it make sense that we begin listening for errors immediately on viewer creation, and not in `handleViewerEvents`?

Reply:
We've confirmed that `handleViewerEvents()` *only* handles the generic `viewerevent`, therefore our initial thoughts are invalid. Merging